hardware/raspberrypi/bootmodes/net_tutorial.md

```diff
@@ -35,6 +35,22 @@ Ensure the output `0x3020000a` is correct.
 
 The client configuration is almost done. The final thing to do is to remove the `program_usb_boot_mode` line from `config.txt` (make sure there is no blank line at the end). You can do this with `sudo nano /boot/config.txt`, for example. Finally, shut the client Raspberry Pi down with `sudo poweroff`.
 
+## Ethernet MAC address
+Before configuring network boot make a note of the serial number and mac address so that the board can be identified by the TFTP/DHCP server.
+
+On Raspberry Pi 4 the MAC address is programmed at manufacture and there is no link between the MAC address and serial number. Both the MAC address and serial numbers are displayed on the bootloader HDMI diagnostics screen.
+
+To find the Ethernet MAC address:  
+# mac address (ip addr) - it should start with DC:A6:32
```

Comment:
"should start with B8:27:EB or DC:A6:32"
(as this page covers all models of Pi, not just Pi4)

Reply:
Probably worth getting rid of `should`, since it's not quite gramatically correct here. Something like `Raspberry Pi MAC addresses start with either B8:27:EB or DC:A6:32`.

Reply:
Since there are multiple OIDs and potentially configurable OIDs I've removed that bit. I've also used the simpler "ethtool -P eth0" command because it's less error prone